addons/io/pom.xml

```diff
@@ -14,12 +14,13 @@
 
 	<packaging>pom</packaging>
 
-	<modules>
-		<module>org.openhab.io.openhabcloud</module>
-		<module>org.openhab.io.hueemulation</module>
-		<module>org.openhab.io.transport.feed</module>
-		<module>org.openhab.io.imperihome</module>
-	</modules>
+  <modules>
```

Comment:
@kaikreuzer @wborn and me interpreted the guidelines that all xml files should contain tabs, so he changed all the pom files and I approved/merged them. Do you agree with this?

General, please change it back to tabs, so that it remains consistent.